### PR TITLE
New: NamespaceInfo Proxy API

### PR DESCRIPTION
**Pull request by ericrpowers**
#### Description
There is interest to add more read-only access to a namespace's information, so instead of making a proxy API per attribute, we will create a new API called `/namespaceinfo` that we will use to house all the requested attributes. This will lead to `/namespacepolicyinfo` to be deprecated.

Values we want for now are:
- PU traffic actions
- Tag Prefixes
- Name
- Description
- Protected

### CODE REVIEW

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "proxy-api-namespaceinfo",
    "component": "backend",
    "pr-id": "1251",
    "commit-sha": "4a7014a50d6b7aa35eb62c6f6603f20605f90668",
    "pipeline": "master"
  },
  {
    "project": "proxy-api-namespaceinfo",
    "component": "gaia",
    "pr-id": "672",
    "commit-sha": "855fe894f64e3dbd4a5d41524da97d544845d419",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "proxy-api-namespaceinfo",
    "component": "backend",
    "pr-id": "1251",
    "commit-sha": "06480329eb04f09db30e9c14298ee6ad728b9e84",
    "pipeline": "master"
  },
  {
    "project": "proxy-api-namespaceinfo",
    "component": "gaia",
    "pr-id": "672",
    "commit-sha": "855fe894f64e3dbd4a5d41524da97d544845d419",
    "pipeline": "master"
  }
]
```

File: specs/namespaceinfo.spec [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - Inherit
+
+  - name: prefixes
+    description: List of tag prefixes that will be used to suggest policies.
+    type: list
+    exposed: true
+    subtype: string
+    read_only: true
+
+  - name: protected
```
Comment (primalmotion) [inline specs/namespaceinfo.spec:43]: should be readonly

**Review by primalmotion — CHANGES_REQUESTED** [review]

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "proxy-api-namespaceinfo",
    "component": "gaia",
    "pr-id": "672",
    "commit-sha": "e6c8a9abfd90360a260f8ee7eb7a17d15dddd2e2",
    "pipeline": "master"
  },
  {
    "project": "proxy-api-namespaceinfo",
    "component": "backend",
    "pr-id": "1251",
    "commit-sha": "8591a166aac6419519edafe13c64f8a3a532a116",
    "pipeline": "master"
  }
]
```

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "proxy-api-namespaceinfo",
    "component": "backend",
    "pr-id": "1251",
    "commit-sha": "34bb6a832ee83c80c4fba258c04c22082f4195fd",
    "pipeline": "master"
  },
  {
    "project": "proxy-api-namespaceinfo",
    "component": "gaia",
    "pr-id": "672",
    "commit-sha": "e6c8a9abfd90360a260f8ee7eb7a17d15dddd2e2",
    "pipeline": "master"
  }
]
```

**Review by primalmotion — APPROVED** [review]

**Pull request merged**